spec/support/vcr_cassettes/vaos/systems/get_facilities_multiple.yml

```diff
@@ -0,0 +1,1334 @@
+---
+http_interactions:
+  - request:
+      method: get
+      uri: https://veteran.apps.va.gov/var/VeteranAppointmentRequestService/v4/rest/direct-scheduling/parent-sites?facility-code=984
```

Comment:
In your request here it doesn't appear as though you are sending an array, what am I missing here?

Reply:
> Note that there's a bug in VCR where it does not record the url correctly when you flatten the url params (don't use []=, which the VAOS service doesn't). However I did test this out intercepting the request and it build the url as VAOS expects outside of tests

Reply:
No matter how many url param array items you pass it only captures the last one in VCR, but I tested it by making a Sinatra service locally and outside of specs it does pass full set of url params correctly.